localstripe/resources.py

```diff
@@ -1308,7 +1309,7 @@ def __init__(self, customer=None, metadata=None, items=None,
         self.canceled_at = None
         self.discount = None
         self.ended_at = None
-        self.quantity = 1
+        self.quantity = items[0]['quantity']
```

Comment:
This is what will need to change when we support plans with different subscriptions. For now I'm leaving it like this as nothing else is prepared to handle that scenario